### DISCUSSION TITLE
Invalid values for Transmit Ticks (part of Tx Status Report)?

### DISCUSSION THREAD

**Discussion by olavt** [zwave-js/zwave-js#5240] [Q&A]
I have an active support case with Silicon Labs about seeing invalid values for Transmit Ticks. The problem I see is that in some cases the value for Transmit Ticks reported are way higher than the elapsed time since the Send Data Command was issued.

I have instrumented my controller to look for these and get errors like this:

Indicated transmission time (4400 ms) is longer than the total execution time for the command (232.6352 ms).

(raw bytes for above message: 01 1d 00 13 5e 00 01 b8 01 ab c9 7f 7f 7f 01 01 05 05 00 00 00 02 0d 05 07 7f 7f 7f 7f 7f 78)

This typically / always occurs after this failure message was received first:

01 1d 00 13 5e 02 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 ad

Is this something you have observed happening on your networks too?

Note: This is not directly related to Z-Wave JS, but about Z-Wave in general.

**Comment by AlCalzone** [comment]
Hey @olavt, sorry for the delay - I'm only slowly catching up with issues from my leave in Q4.

I haven't noticed this before, but I also haven't looked for it. One log where I knew this failure occurred too does not show the issue - the next message takes 1 tick (10ms).
